plugins/aws-catalog-backend/CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,119 @@
+# Code of Conduct
```

Comment:
I've removed all duplicates (including those from the awards plugins because why not) via 961794d

Reply:
hmmm, i wonder if we should separate this to a separate PR since our semantic release tooling may try to bump the awards packages based on files change (and a commit type that signifies a version bump)

Reply:
Sure, I've restored those files via commit 24b21dd